src/Components/Quotes.js

```diff
@@ -0,0 +1,16 @@
+import styles from '../styles/quotes.module.css';
+// import Header from './Header';
+
+const Quotes = () => (
+  <div>
+    {/* <Header /> */}
```

Comment:
Please kindly remove commented code if its not being used in this file for it makes your code base look untidy.